src/sync/fence.rs

```diff
@@ -114,6 +114,7 @@ pub struct Fence<T = ()> {
     value: Option<T>,
     poll_rate: Duration,
     handle: vk::Fence,
+    wait_thread_spawned: bool,
```

Comment:
If we switch to this approach, the `poll_rate` field and related functions should be removed